src/main/java/net/minecraftforge/event/world/RaidEvent.java

```diff
@@ -0,0 +1,81 @@
+package net.minecraftforge.event.world;
+
+import net.minecraft.entity.monster.AbstractRaiderEntity;
+import net.minecraft.world.raid.Raid;
+import net.minecraftforge.eventbus.api.*;
+
+/**
+ *  RaidEvent is the base class of all Raid events.
+ */
```

Comment:
There is no specification in any of these events for what canceling the event does. 
Is the root event cancelable? What does canceling the raid member event do? Documentation is severly lacking.